ext/skeleton/config.m4.in

```diff
@@ -21,23 +21,12 @@ PHP_ARG_ENABLE([%EXTNAME%],
 if test "$PHP_%EXTNAMECAPS%" != "no"; then
   dnl Write more examples of tests here...
 
-  dnl # get library FOO build options from pkg-config output
-  dnl AC_MSG_CHECKING(for libfoo)
-  dnl if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists foo; then
-  dnl   if $PKG_CONFIG foo --atleast-version 1.2.3; then
```

Comment:
For some libraries it might be useful to check minimum required version. So this one might be good to have as a demo how to integrate it a bit...

```
dnl   if `$PKG_CONFIG libfoo --atleast-version=1.2.3`; then
```

Reply:
Taking this back. It is better and nicer to use this, so this is all good to go, as done, yes.

```
PKG_CHECK_MODULES([FFI], [libffi >= 3.2.1])
```